src/beans/Section.ts

```diff
@@ -51,14 +111,18 @@ class Section {
       })
     );
     this.instructors = unique(
-      this.meetings.reduce(
-        (instructors, meeting) => [...instructors, ...meeting.instructors],
-        []
-      )
+      this.meetings
+        .map<string[]>(({ instructors }) => instructors)
+        .reduce((accum, instructors) => [...accum, ...instructors], [])
```

Comment:
TypeScript doesn't recognize a reduce with different types for the source array & accumulator (as in `Array<A>.reduce<B>((accum: B, elem: A) => B, start: B): B`) unfortunately, so we just move the transformation to an explicit map step and then reduce the `string[][]` with a `string[]` accumulator.